### PR TITLE
Revise Multi-storage Transaction Sample

### PR DESCRIPTION
**Pull request by brfrn169**
This PR revises the Multi-storage Transaction Sample. I'll add inline comments for the details. Please take a look!

### CODE REVIEW

File: multi-storage-transaction-sample/README.md [hunk 1]
```diff
@@ -123,35 +125,43 @@ scalar.db.multi_storage.namespace_mapping=customer:mysql,order:cassandra,coordin
...
 
-### Initial data
+## Load schema
+
+You then need to apply the schema with the following command.
+Please download the schema tool `scalardb-schema-loader-<version>.jar` that can be found in [releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB.
+
+```shell
+$ java -jar scalardb-schema-loader-<version>.jar --config database.properties --schema-file schema.json --coordinator
+```
```
Comment (brfrn169) [inline multi-storage-transaction-sample/README.md:154]: The most major change is here.

Previously, we loaded the schema using the docker-compose command. However, I've modified the process to download the schema loader jar and load the schema using that instead.

File: multi-storage-transaction-sample/README.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Multi-storage Transaction Sample
 
-This is a sample application for Multi-storage Transaction in ScalarDB.
+This tutorial describes how to create a sample application by using ScalarDB with Multi-storage Transactions.
```
Comment (brfrn169) [inline multi-storage-transaction-sample/README.md:3]: The other changes are refinements in wording.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

File: multi-storage-transaction-sample/README.md [hunk 3]
```diff
@@ -99,14 +101,14 @@ The ER diagram for the schema is as follows:
...
-3. Placing an order. An order is paid by a credit card. It first checks if the amount of the money of the order exceeds the credit limit. If the check passes, it records order histories and updates the `credit_total`
-4. Getting order information by order ID
-5. Getting order information by customer ID
-6. Repayment. It reduces the amount of `credit_total`.
+2. Placing an order by credit card (checks if the cost of the order is below the credit limit, then records order history and updates the `credit_total` if the check passes)
+3. Getting order information by order ID
+4. Getting order information by customer ID
+5. Repayment (reduces the amount in the `credit_total`)
 
 ### Configuration
```
Comment (brfrn169) [inline multi-storage-transaction-sample/README.md:109]:
```suggestion
## Configuration
```

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**